modules/docs/arrow-docs/docs/docs/arrow/typeclasses/applicative/README.md

```diff
@@ -98,6 +98,23 @@ Computation happens when `.value()` is invoked.
 Option.applicative().run { Some(1).map2Eval(Eval.later { Some("x") }, { z: Tuple2<Int, String> ->  "${z.a}${z.b}" }).value() }
 ```
 
+#### Kind<F, A>#followedBy / followedByEval
+
+Sequences actions, discarding the value of the first argument.
+The [`Eval`]({{ '/docs/apidocs/arrow-core-data/arrow.core/-eval' | relative_url }}) variant allows you to pass lazily calculated values.
+
+```kotlin:ank
+Option.applicative().run { Some(1).followedBy(Some(2)) }
+```
+
+#### Kind<F, A>#apIgnore / apIgnoreEval
+
+This is a reverse for `followedBy`. Sequences actions but discarding the value of the second argument.
+
+```kotlin:ank
+Option.applicative().run { Some(1).apIgnore(Some(2)) } 
+```
+
```

Comment:
This should be under `Apply` shouldn't it?

Reply:
In fact, all the documented methods in this file belong to Apply, there is a section explaining what Apply is just below this fragment. I can just copy these two at the Apply docs too 🤔

Reply:
Ah ok, then leave it as it is. Thats probably less confusing